modules/rdma-creating-shared-device-rdma-roce.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-creating-shared-device-rdma-roce_{context}"]
+
+= Creating a shared device RDMA on RoCE
+
+This procedure describes how to create the workload pods for a shared device RDMA for the NVIDIA Network Operator and how to test the pod configuration.
+
+The NVIDIA GPUDirect RDMA device is shared among multiple pods on the {product-title} worker node where the device is exposed. 
+
+.Prerequisites
+
+* Ensure that the Operator is running.
+
+* Delete the `NicClusterPolicy` custom resoure (CR), if it exists. 
+
+.Procedure
+
+. Generate a custom pods resource:
++
+[source,yaml]
+----
+$ cat <<EOF > rdma-eth-32-workload.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-eth-32-workload
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: rdmashared-net
+spec:
+  nodeSelector: 
+    kubernetes.io/hostname: nvd-srv-32.nvidia.eng.rdu2.dc.redhat.com
+  serviceAccountName: rdma
+  containers:
+  - image: quay.io/redhat_emp1/ecosys-nvidia/gpu-operator:tools
```

Comment:
Replace this quay with `- image: quay.io/edge-infrastructure/nvidia-tools:0.1.5`

Reply:
Replaced.